ui/tests/acceptance/mfa-login-test.js

```diff
@@ -35,7 +35,7 @@ module('Acceptance | mfa-login', function (hooks) {
   });
 
   const login = async (user) => {
-    await visit('/vault/auth');
+    await visit('/vault/auth?with=token');
```

Comment:
MFA tests were failing since the auth form had different tabs and it was defaulting to the oidc tab, so this enforces that it will go to the auth form with the dropdown of auth methods.

Reply:
🤔 This means something wasn't clearing out in a previous test correctly. I wonder what changed!

Reply:
Hmm I'm curious too! I might explore it more after finishing up these tickets :D